successstories/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167873271-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+
+        gtag('js', new Date());
+        gtag('config', 'UA-167873271-1');
+    </script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="Over the last few years of delivering SEF initiated programs, we noticed one of the biggest challenges faced by undergraduate
+                            students was their ability to speak in English. While their reading and writing skills were very good, their level of spoken
+                            English was not on par. These were due to a number of reasons including inadequate resources to practice spoken English in
+                            a safe environment and focus on mastering academic English using rigid patterns not transferrable when speaking">
```

Comment:
maybe we can get this from the scholarX page

Reply:
Yes. will do.